apps/server/lib/google-drive/src/internal/qs/LICENSE.md

```
@@ -0,0 +1,24 @@
+BSD 3-Clause License
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)